### DISCUSSION TITLE
Add an onUpdated event

### DISCUSSION THREAD

**Discussion by WilliamStam** [orchidjs/tom-select#284] [Ideas]
right now we have onChange / change event but that gets fired on every selection of a multi select. 

so say you want to run an event like "lookup some data on change" it becomes a bit more difficult since its basicaly going to run on every selection. 

so then i got to try onDropdownClose but that fires before the change with a normal single select. 

proposing adding in an event that checks the current value, if its different to to the "new" value, then run the callback when the dropdown closes.

**Comment by WilliamStam** [comment]
```
 onChange: (v)=>{
            console.log("onChange",v)
            // value.value = v;
        },
        onDropdownClose: ()=>{
            console.log("onDropdownClose")
            // emit('update:modelValue',value.value)
        }
```

for a multi select, selecting 3 items

```
onChange ['4']
onChange (2) ['4', '43']
onChange (3) ['4', '43', '10']
onDropdownClose
```

when its a single select

```
onDropdownClose
onChange 2
```

(not sure if this is a bug or not) it just gets kinda hard to "run a function when this have changed" in a sane way like this i think.

**Reply by oyejorge** [reply]
Looks like a bug to me. #288
